com.unity.render-pipelines.high-definition/Documentation~/Custom-Post-Process.md

```diff
@@ -333,3 +279,5 @@ If your effect does not display correctly:
 * Check that your effect's Shader compiles and that the reference to the Material in your post process Volume is not null.
 
 * In the Volume that contains your post process, make sure that it has a high enough priority and that your Camera is inside its bounds.
+
+* Checks that your shader doesn't contains any **clip()** instruction, that the blend mode is set to Off and the output alpha is always 1.
```

Comment:
Check that your shader doesn't contain any **clip()** instructions, that the blend mode is set to **Off** and the output alpha is always **1**.